docs/data/versions.toml

```diff
@@ -24,10 +24,10 @@ url= "/download/postgresql-42.2.27.jre6.jar"
 # Past Versions
 
 [[past]]
-v_name= "Postgresql JDBC 42.5.1"
-version= "42.5.1"
+v_name= "Postgresql JDBC 42.5.3"
+version= "42.5.3"
```

Comment:
Is it intentional that you hide 42.5.1 and 42.5.2 versions?
Should we add entries rather than update old ones?